### ISSUE TITLE
Make it more clear where to report problems or bugs on contrib modules

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#832]
We have a link on every project page to the relevant issue queue. BUT, not all users may understand that the issue queue is where they should report bugs and problems. 

Why not put a label on this field telling users that this is where they report bugs or problems with the module. 

![issue queue](https://user-images.githubusercontent.com/3144571/136095040-eb92076d-b41a-4985-83c7-ef64931ec1f1.png)

**Comment by bugfolder** [comment]
This is similar to [something I'd proposed here](https://github.com/backdrop-ops/backdropcms.org/issues/627#issuecomment-858132573).

**Comment by stpaultim** [comment]
Absolutely. Let's leave this ticket open to hightlight this specific problem, since the other issue is about a bunch of things. If we fix this in that issue, we can close this later.

This came about as I was showing this page to Sarvasri, who pointed out that it wasn't at all clear to him that the Issue Queue is where you report bugs.
